packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -1,42 +1,421 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
```

Comment:
The changes in this file are just copy/pasting `dataviews-patterns.js`. These can be skipped from the review.

Reply:
I was surprised that GitHub didn't say the file moved.

Reply:
There are changes in other files (imports), not sure if that's the reason.